bundles/config/org.eclipse.smarthome.config.discovery/src/main/java/org/eclipse/smarthome/config/discovery/internal/console/InboxConsoleCommandExtension.java

```diff
@@ -100,13 +102,37 @@ public void execute(String[] args, Console console) {
                 case SUBCMD_CLEAR:
                     clearInboxEntries(console, inbox.getAll());
                     break;
+                case SUBCMD_REMOVE:
+                    if (args.length > 1) {
+                        List<DiscoveryResult> results = Stream.of(new DiscoveryResult[] {})
```

Comment:
A simple `new ArrayList()` would enhance readability.

Reply:
It would also make sure that the `results` is not immutable...

Reply:
Ok I can change that.